presto-native-execution/presto_cpp/main/http/filters/JwtTokenFilter.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <proxygen/httpserver/Filters.h>
+#include <proxygen/httpserver/RequestHandlerFactory.h>
+
+namespace facebook::presto::http::filters {
+
+class JWTTokenFilter : public proxygen::Filter {
+ public:
+  explicit JWTTokenFilter(proxygen::RequestHandler* upstream)
+      : Filter(upstream), requestRejected_(false) {}
+
+  void onRequest(std::unique_ptr<proxygen::HTTPMessage> msg) noexcept override;
```

Comment:
It will be great to add documentation to each of these methods. If a function is pass-through, let's mention that.

Reply:
I made a comment. The doc is found in the header files that implement these functions for the filter (and request handler). So I didn't want to copy that in. Looking at the other filters no explanation is given.
However, I agree it is no easy to understand what is happening and I spent quite a bit of time so I added a summary of what is happening.